pkg/cache/db/queries.go

```diff
@@ -64,5 +74,32 @@ const (
 		FROM descendants
 	`
 
-	healthyComponentsRatio = `SELECT CAST(AVG(health = 0) * 100 as INTEGER) as ratio FROM Component`
+	clusterHealthScore = `SELECT CAST(AVG(health = 0) * 100 as INTEGER) as score FROM Component`
+
+	setPod = `
+		INSERT INTO Pod (
+			name,
+			namespace,
+			uid,
+			node,
+			createdAt
+		) VALUES (
+			?,
+			?,
+			?,
+			?,
+			?
+		) ON CONFLICT(uid) DO UPDATE SET
+			name = excluded.name,
+			namespace = excluded.namespace,
+			node = excluded.node,
+			createdAt = excluded.createdAt
+	`
+
+	nodeStatistics = `
+	SELECT node, COUNT(*)
+	FROM Pod
+	WHERE datetime(createdAt) < datetime('now', '-5 minutes')
+	GROUP BY node
+	`
```

Comment:
Do we want to change the query so it will return nodes that have no pending pods created more than 5 minutes ago as well?